nemo/collections/nlp/data/language_modeling/megatron/dataset_utils.py

```diff
@@ -721,6 +743,7 @@ def get_samples_mapping(
     # This should be a barrier but nccl barrier assumes
     # device_index=rank which is not the case for model
     # parallel case
```

Comment:
Clean up comments

Reply:
Done. And found two distributed barriers instead of one for some reason.